torch/onnx/_internal/fx/decomposition_table.py

```diff
@@ -51,7 +51,11 @@ def _create_onnx_supports_op_overload_table(
                 continue
 
             exporter_look_up_key = op_overload_packet._qualified_op_name
-            if registry.get_function_group(exporter_look_up_key) is None:
+            # overload name is torch.ops.<domain>.<op_name>.<overload>
+            domain, opname = exporter_look_up_key.split("::")
+            if not registry.is_registered_op(
+                domain=domain, op_name=opname, overload=None
+            ):
                 # This aten op doesn't have ONNX overloads.
```

Comment:
I think I added that. lol

A weird thing I found about this code is that if we try to check the membership inside aten overload level (current code check on overloadpacket level), we will let more aten ops get into our table, and eventually fails on some ops.

Reply:
To check my understanding

> try to check the membership inside aten overload level

This is the right thing to do right?

> eventually fails on some ops

These are real bugs right?

Reply:
Not sure, I think this code lives here since the first draft. I recall there will be 20 more overloads registered into decomposition table resulting different FX graph. I can look into those ops.

Reply:
Revisit on this. The main issue on this is that "we need default overload fallback.". Based on what I have found during decomposition table creation. The thing is that we could have much less ATen overloads than PyTorch, but the original logic basically says as long as default overload is supported, the overload should be included into table. This part is fat from torchlib, so I guess it's some ATen decompose knowledge. I refactored this part to accept default overload fallback, and delete the rely on overloadpacket. It should be the same thing.